Fabric.Identity.API/scripts/Register-Identity-IdPSS.ps1

```diff
@@ -37,9 +39,15 @@ if($null -ne $tenants) {
             -tenantId $tenant `
             -clientSecret $clientSecret `
             -clientId $clientId `
-            -installConfigPath $installConfigPath
+            -installConfigPath $installConfigPath `
+			-appName $appName
 
         # Manual process, need to give consent this way for now
         Start-Process -FilePath  "https://login.microsoftonline.com/$tenant/oauth2/authorize?client_id=$clientId&response_type=code&state=12345&prompt=admin_consent"
     }
-}
\ No newline at end of file
+}
+
+#identity registration
+Register-Identity -appName "Identity Service" -replyUrls $replyUrls -configSection $installSettingsScope -installConfigPath $installConfigPath
```

Comment:
Should we add a command line parameter so the user can decide to register idpss or identity separately?

Reply:
The reason I ask is because we only need to register Identity in one of the Azure tenants. IdPSS needs to be registered in all tenants that are in the environment...so it would be nice to either allow only IdPSS or both IdPSS and Identity.

Reply:
Yah, currently I use the claimsissuer as the tenant used to register identity. I think prompting them for (claimsissuer and allowedtenants) during the identity registration, would make sense for scenario two, and no nodes are added for scenario one when Idpss is registered.

Reply:
added switches to cover this, and removed all the node creation and hard coded the necessary install.config nodes, since they will need to manually enter all tenant info.